src/cmdline.rs

```diff
@@ -13,30 +13,38 @@
 // limitations under the License.
 
 use crate::errors::*;
+use clap::{ArgEnum, ArgGroup, StructOpt};
 use std::convert::{TryFrom, TryInto};
 use std::env;
 use std::ffi::OsString;
 use std::path::PathBuf;
-use structopt::{
-    clap::{arg_enum, AppSettings, ArgGroup},
-    StructOpt,
-};
 use strum::{EnumVariantNames, VariantNames};
 
-arg_enum! {
-    #[derive(EnumVariantNames, StructOpt, Debug)]
-    #[allow(non_camel_case_types)]
-    pub enum StatsFormat {
-        text,
-        json
+#[derive(Copy, Clone, EnumVariantNames, ArgEnum, StructOpt, Debug)]
+#[allow(non_camel_case_types)]
```

Comment:
Can we avoid this by using a annotation for the enum varaints instead? https://github.com/clap-rs/clap/blob/v3.1.0/examples/git-derive.md suggests (and the linked source) suggest we can use camel case types, with non-camel case cli args.

Reply:
done